README.md

```diff
@@ -55,14 +55,65 @@ The source code in this package is released under the Apache-2.0 License. For fu
 The [3rdparty-licenses.txt](3rd-party-licenses.txt) contains a list of other open source components included in this package.
 
 
-## Installation & Usage as a ROS package
+## Installation & Usage as a [ROS2] package
 
 ### Dependencies
 This package depends on the linear algebra library [Eigen], which can be installed by
 
     sudo apt-get install libeigen3-dev
 
-The [ROS] dependencies are listed in the package.xml and can be installed by
+The [ROS2] dependencies are listed in the package.xml and can be installed by
+
+    rosdep install steering_functions
+
+
+### Building
+
+To build this package from source, clone it into your workspace and compile it in *Release* mode according to
+
+    cd ros2_ws/src
+    git clone https://github.com/hbanzhaf/steering_functions.git
+    cd ..
+    colcon build --packages-up-to steering_functions --cmake-args -DCMAKE_BUILD_TYPE=Release
+
+Note: the demo node is not yet ported to [ROS2], only the library functions are available.
+
+
+### Linking
+
+To link this library with another [ROS2] package, add these lines to your package's CMakeLists.txt
+
+    find_package(steering_functions REQUIRED)
+
+    target_link_libraries(your_target steering_functions::steering_functions)
```

Comment:
What is your_target?

Reply:
When you create your own target and want it to link to steering functions, this is the way to do it. Replace your_target with the actual target name you are using.